packages/next/pages/_document.tsx

```diff
@@ -617,11 +621,10 @@ export class NextScript extends Component<OriginProps> {
         data-next-page={page}
         key={page}
         src={
-          assetPrefix +
-          (dynamicBuildId
+          assetPrefix + dynamicBuildId
```

Comment:
This is missing `encodeURI`.

Reply:
it doesn't need to be touched. file name goes through ```getPageFile``` method which already cares about symbols in file path, line 742